### DISCUSSION TITLE
What should be the default verbosity?

### DISCUSSION THREAD

**Discussion by tomkerkhove** [arcus-azure/arcus.templates#355] [Q&A] [answered]
Currently we use debug as default verbosity:
https://github.com/arcus-azure/arcus.templates/blob/master/src/Arcus.Templates.WebApi/Startup.cs#L313

But what is the ideal default verbosity?

**Comment by tomkerkhove** [comment, marked as answer]
I think having it on `Information` is a good starting point

**Reply by stijnmoreels** [reply]
With #359 we have set it to `Information` and to `Debug` when the configuration is `DEBUG`. But @fgheysels thinks about always setting this to `Information`.
Thoughts?

**Reply by tomkerkhove** [reply]
Both are fine by me

**Reply by fgheysels** [reply]
Set it to Information and put it in the Configuration
https://github.com/arcus-azure/arcus.templates/discussions/368

**Reply by stijnmoreels** [reply]
Done in #372
